Dockerfile.backwards

```diff
@@ -7,6 +7,8 @@ RUN echo "yaml https://s3-us-west-2.amazonaws.com/rosdep/base.yaml" > /etc/ros/r
 COPY . /opt/package
 WORKDIR /opt/package
 
+RUN apt update || apt install -y curl && curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
```

Comment:
Did you mean `apt update && apt install`?
This will run `apt install` if `apt update fails instead of short-circuiting on `apt update`.